Milestone 6:  Set accessibility

s2jack opened the pull request:
**Page Title:** No issue found

**Image text alternatives:**  There is no "img" element

**Text Headings:** p element changed  to h2 element in contact form section
 https://github.com/s2jack/create-portfolio-website/commit/45b5b0b268a14f8ff857550096a8e8d153ca9203#r98763034

**Color Contrast:** links color property and decoration property changed in footer section
https://github.com/s2jack/create-portfolio-website/commit/45b5b0b268a14f8ff857550096a8e8d153ca9203#r98766401

**Resize:** we didn't figure out how can we do it. When I set my browser text size to larger, there was no change.

**Interaction:** Add labels to inner form elements
https://github.com/s2jack/create-portfolio-website/commit/45b5b0b268a14f8ff857550096a8e8d153ca9203#r98767446

**Moving Content:** No issues found

**Multimedia:** there is no multimedia content

**The Basic Structure of the Page**: No issues found

s2jack committed:
Edit accessibility properties
Add label to form elements
Add inner text to Link elements
Change headings tags to h2

s2jack committed:
Fix linter errors

Fix linter errors

s2jack committed:
Fix linters

Fix linters

Sonikak004 committed:
Fix linter errors

Zuhaib042 commented on index.html:
```diff
@@ -206,10 +204,10 @@ <h3 class="ulheader">Skills</h3>
-                <input class="st-input" type="email" name="email" placeholder="Email address" required>
-              </div>
-              <textarea name="message" type="text" maxlength="500" placeholder="Enter your message here..." required></textarea>
+                <label id="name_inp">Your name<input class="st-input" type="text" name="name" placeholder="Your name" maxlength="30" required></label>
+                <label id="email_inp">Your email<input class="st-input" type="email" name="email" placeholder="Email address" required></label>
+              </div> 
+              <label id="message_inp">Your message<textarea name="message" type="text" maxlength="500" placeholder="Enter your message here..." required></textarea></label>
```
- [x] Kindly consider adding a `for` attribute in your label that would have a corresponding `id` in its `input` due to follow best practices. please check out this link for more [info](https://www.w3schools.com/tags/tag_label.asp)

Zuhaib042 reviewed (changes requested):
Hi @s2jack , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![close](https://media.giphy.com/media/BM10SaUSoT789SJkvf/giphy.gif)

## Highlights
✅ Professional PR
✅ Correct Github flow

## Required Changes ♻️
- [x] I think if you wrap your projects' cards main heading in  `h` tag, that would have been better for best accessibility practices. 
- [x] In your about section, there are some issues needs to be addressed in order to proceed, I cannot comment using inline-feature because you haven't added those lines in your changes. those issues are the following:
  - Kindly consider using the `h` tag in `languages` heading. 
  - Kindly consider fixing the contrast ratio for `frameworks` and `skills` headings and all the `li`s for these two and `languages` cards. kindly check out this following pictures for more info. you can install this chrome extension to check the contrast ratio and then adjust it.
![qwer](https://user-images.githubusercontent.com/96683843/215840767-09e7ceaf-b281-4020-accb-f5b6e2fbe2c1.jpg)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Set accessibility
Add projects headers into h element
Add languages header into h element
Add for attribute into label elements

s2jack committed:
Fix linter errors

s2jack commented:
Hey dear @Zuhaib042 , thank you for your time. 
Unfortunately, I must let you know the contrast problem that you mentioned above is false. In my opinion, there is confusion due to the background color. 
![Ekran görüntüsü 2023-02-01 123801](https://user-images.githubusercontent.com/67601435/216008830-1638ed0d-3f3c-4b8a-b264-f9eb979dc37a.png)
As you can see in the picture I posted, the background color seems white. Because there is no background-color attribute. There is an image. And there is no contrast issue. White texts in flat colors are the most readable texts.

frank1738 reviewed (approved):
Hi @s2jack , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media4.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>

## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
